pallets/loans/src/tests.rs

```diff
@@ -16,6 +16,7 @@
 
 use frame_support::{assert_noop, assert_ok};
 use primitives::{BLOCK_PER_YEAR, RATE_DECIMAL};
+use sp_runtime::traits::One;
```

Comment:
This is not used in your changes.

Reply:
@kaichaosun I am not sure. When I did `cargo update`, the current master branch started failing and the compiler was suggesting this import.

Reply:
I think it's necessary to have it